src/client/render.tsx

```diff
@@ -71,9 +85,54 @@ export class CellOutput extends React.Component<ICellOutputProps> {
         // Hack, use same classes as used in VSCode for images (keep things as similar as possible).
         // This is to maintain consistently in displaying images (if we hadn't used HTML).
         // See src/vs/workbench/contrib/notebook/browser/view/output/transforms/richTransform.ts
+        const saveAs = () => {
+            if (this.props.ctx.postMessage) {
+                this.props.ctx.postMessage({
+                    type: 'saveAs',
+                    outputId: this.props.outputId,
+                    mimnType: this.props.mimeType
```

Comment:
Still wondering why `mimnType` is used here and in https://github.com/microsoft/vscode-jupyter/pull/6235. I initially thought it was a typo.

Reply:
We need to know what the user is trhing to save (e.g. if they're trying to save the `png` mime type of not.
This way we don't assume what's being saved.